### ISSUE TITLE
Trouble with quality of results using pretrained model

### ISSUE THREAD

**Issue by poinne** [JiahuiYu/generative_inpainting#549]
I used a pre-trained model for testing, but the results are very poor, and I haven't made any changes. Why would this happen?
it is real image:
![image](https://github.com/JiahuiYu/generative_inpainting/assets/162424423/d01de44a-2d6e-4d90-92ca-cb5bcf2cb05d)
it is mask:
![image](https://github.com/JiahuiYu/generative_inpainting/assets/162424423/713ba5a4-851e-4b8a-9637-12f39b8e3045)
it is result:
![image](https://github.com/JiahuiYu/generative_inpainting/assets/162424423/08b33c30-cb2e-48b9-98cb-d54683541890)

**Comment by jduyou** [comment]
Hello, can you share with me the pre-training model you have downloaded? I can't open the official download link.
